lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -68,17 +71,19 @@
                      this.updateEnrollBtn();
                  },
 
-                 toggleSessionSelectionPanel: function() {
```

Comment:
Note for reviewers - simply moved this function later in the file, did not change it.